terraform/modules/helm/secor/secor-helm-chart/values.yaml

```diff
@@ -13,7 +13,7 @@ namespace: "secor"
 storageClass: "gp2"
 secor_env: "dev"
 image_repository: "sanketikahub/secor"
-region: "us-east-2"
+region: ""
```

Comment:
This required manual changes, right? if users are trying to deploy with just the helm chart?
Can we add some comments here saying required to define a region value with an example?

Reply:
Sure